static/coffee/controllers.coffee

```diff
@@ -573,12 +588,12 @@ controllers.controller('CaseController', ['$scope', '$window', '$timeout', 'Case
   #----------------------------------------------------------------------------
 
   $scope.onWatch = () ->
-    UtilsService.confirmModal("Receive notifications for activity in this case?").then(() ->
+    UtilsService.confirmModal("Receive alerts for activity in this case?").then(() ->
```

Comment:
Thanks for taking a look. I've created #127 for this, will make this change as a separate PR.